Mathport/Syntax/Translate/Basic.lean

```diff
@@ -682,30 +793,30 @@ def trNotation (n : Choice) (args : Array (Spanned Arg)) : M Syntax := do
   match ← getNotationEntry? n.getString!, args.map (·.kind) with
```

Comment:
I think you want to use `args` instead of `args.map (·.kind)` here and propagate the spans below

Reply:
Yes!